aip/auth/4110.md

```diff
@@ -0,0 +1,153 @@
+---
+aip:
+  id: 4110
+  scope: auth
+  state: approved
+  created: 2018-06-24
+  updated: 2019-06-24
+permalink: /auth/4110
+redirect_from:
+  - /4110
+---
+
+# Application Default Credentials
+
+Google auth libraries use a strategy called _Application Default Credentials
```

Comment:
Maybe linkify instead of italicize?

Reply:
We currently do not have a public dev page for ADC. @broady and I are planning to add one.

Reply:
@broady Such a page would be very helpful. 
We have one page here:
https://cloud.google.com/docs/authentication/production
All authorization methods are not described there. All methods are described on this page, but this page lacks other information. :-/ 
https://google-auth.readthedocs.io/en/latest/reference/google.auth.html